samples/dashboards/demo/climate/demo.html

```diff
@@ -6,12 +6,12 @@
 <script src="https://code.highcharts.com/highcharts.js"></script>
 <script src="https://code.highcharts.com/highcharts-more.js"></script>
 <script src="https://code.highcharts.com/modules/accessibility.js"></script>
-<script src="https://code.highcharts.com/modules/data-tools.js"></script> <!-- @todo replace with math module -->
 <script src="https://code.highcharts.com/modules/map.js"></script>
```

Comment:
want to fix this also?

Reply:
Is it not included in the math-modifier.js ?

Reply:
I think @goransle meant `modules/map`, right?

Reply:
Mea culpa, you are right!

Reply:
Thanks, fixed map & stock.